### PR TITLE
LineSegements2: Add a "threshold" Raycaster param

### PR DESCRIPTION
**Pull request by gkjohnson**
Adds a threshold raycaster param like Lines uses so line segments 2 can be expanded by a pixel width for raycasting.

```js
const raycaster = new Raycaster();
raycaster.params.Line2 = { threshold: 2 };

// raycast...
```

ping @WestLangley

### CODE REVIEW

File: examples/js/lines/LineSegments2.js [hunk 1]
```diff
@@ -70,14 +70,21 @@ THREE.LineSegments2.prototype = Object.assign( Object.create( THREE.Mesh.prototy
 
 			}
 
+			var threshold = 0;
+			if ( 'Line2' in raycaster.params ) {
+
+				threshold = raycaster.params.Line2.threshold || 0.0;
+
+			}
```
Comment (WestLangley) [inline examples/js/lines/LineSegments2.js:78]: Does this pattern work?
```js
var threshold = ( raycaster.params.Line2 !== undefined ) ? raycaster.params.Line2.threshold || 0 : 0;
```

Comment (gkjohnson) [reply examples/js/lines/LineSegments2.js:78]: It does -- I'd initially considered that but thought it was more difficult to read. If you'd prefer I can change it.

Comment (WestLangley) [reply examples/js/lines/LineSegments2.js:78]: It is not up to me, but that is the pattern we usually follow. /ping @Mugen87

Comment (gkjohnson) [reply examples/js/lines/LineSegments2.js]: Got it -- done!

**Review by WestLangley — APPROVED** [review]

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**